### PR TITLE
added experience search feature

### PR DESCRIPTION
**Pull request by rohit9210**
added search feature for experience dashboard, using mock data. The feature can be later refined while API integration @idougwarner @leomanza

### CODE REVIEW

File: src/hooks/api/experiences.ts [hunk 1]
```diff
@@ -14,4 +14,34 @@ export const useGetAllExperiences = () => {
         experiences: data,
         isSearching: isLoading,
     };
+};
+
+/** Terporary util function to mock search functionality, will be removed when API is implemented */
```
Comment (leomanza) [inline src/hooks/api/experiences.ts:19]: typo _Terporary_

File: src/services/experiences.ts [hunk 2]
```diff
@@ -4,13 +4,23 @@ import { EXPERIENCES } from "../mock/mock";
...
         //   .then((response) => response.data);
         return new Promise(function (resolve) {
             return setTimeout(() => resolve(EXPERIENCES.other), 100);
             // return setTimeout(() => reject(), 1500);
         });
     },
+
+    searchExperience: async (): Promise<Array<any>> => {
+        // return axios
+        //   .get(`/expereinces`)
```
Comment (leomanza) [inline src/services/experiences.ts:17]: typo _expereinces_

**Review by leomanza — CHANGES_REQUESTED** [review]
looks good. some typos

File: src/hooks/api/experiences.ts (hunk 1 above)
Comment (rohit9210) [reply src/hooks/api/experiences.ts]: @leomanza typo fixed

File: src/services/experiences.ts (hunk 2 above)
Comment (rohit9210) [reply src/services/experiences.ts]: @leomanza  typo fixed

**Review by fimak — APPROVED** [review]
👍

**Pull request merged**